src/lib/style/base/_form.scss

```diff
@@ -1,24 +1,18 @@
 @use '../abstracts/variables' as SRC;
 
 .bootstrap-4-backport {
-
   label {
     color: SRC.$gray-formtext;
     font-weight: bold;
   }
 
-  input[type=text],
-  input[type=password],
-  input[type=search],
-  textarea {
-    &.form-control {
-      margin-bottom: 1rem;
-      border: 1px solid SRC.$border-color-gray;
-      padding: 12px 13px;
-      height: auto;
-      border-radius: 2px;
-      box-shadow: none;
```

Comment:
Refactored most of these properties to be Sass variables so Bootstrap consistently applies them.